src/components/Match/MatchLog.jsx

```diff
@@ -96,7 +96,11 @@ const logColumns = [heroTdColumn, {
     switch (row.type) {
       case 'kills': {
         const hero = heroNames[row.detail] || {};
-        return <img src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />;
+        const comp = [<img key="hero" src={`${API_HOST}${hero.img}`} className={styles.imgSmall} role="presentation" />];
+        if (row.tracked_death) {
+          comp.push('Tracked');
+        }
```

Comment:
I don't know how to improve it, something like [this](http://imgur.com/kZLjsHb) maybe (that I have no idea how to do it), or remove it from log at all.